tools/vagrant/Vagrantfile

```diff
@@ -128,12 +132,12 @@ INIT_DB
 $script_end = <<SCRIPT
   set -e
   set -x
-
+  echo "`date`: build-deploy-start" >> /tmp/vagrant-times.txt
   HOME=/home/vagrant
   source /etc/environment
   # Build OpenWhisk using gradle
   cd ${OPENWHISK_HOME}
-  su vagrant -c './gradlew distDocker --parallel'
+  su vagrant -c './gradlew  tools:cli:distDocker -PdockerImagePrefix=openwhisk --parallel'
```

Comment:
I don't think we need `--parallel` here since only one project, the CLI, is being built.